packages/server/graphql/types/TeamPromptResponse.ts

```diff
@@ -0,0 +1,45 @@
+import {GraphQLFloat, GraphQLID, GraphQLNonNull, GraphQLObjectType, GraphQLString} from 'graphql'
+import {GQLContext} from '../graphql'
+import Reactable, {reactableFields} from './Reactable'
+import Team from './Team'
+
+const TeamPromptResponse: GraphQLObjectType = new GraphQLObjectType<any, GQLContext>({
+  name: 'TeamPromptResponse',
+  description: 'A response of a single team member in a team prompt',
+  interfaces: () => [Reactable],
+  fields: () => ({
+    ...reactableFields(),
+    userId: {
+      type: new GraphQLNonNull(GraphQLID),
+      description: 'Id of the user who created the team prompt response'
+    },
+    user: {
+      type: require('./User').default,
+      description: 'The user who created the response',
+      resolve: ({userId}: {userId: string}, _args: unknown, {dataLoader}) => {
+        return dataLoader.get('users').load(userId)
+      }
+    },
+    content: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: 'the content of the response'
+    },
+    plaintextContent: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: 'the plain text content of the response'
+    },
+    sortOrder: {
```

Comment:
-1 we don't have that field in DB so it won't resolve now.

Reply:
we do, I've added that per your suggestion: https://github.com/ParabolInc/parabol/pull/6068/files#diff-17d65a8dd63bfe43cfadd39f2b9814637352fff193e0cbd8ce3ee5c0d2bd0c53R24 😁

Reply:
+1 safe to keep a float for now, but in the future we'll probably want to create a new type which is equivalent to `Number.MAX_SAFE_INTEGER` so we don't have to validate that it's an integer